policyengine_us/parameters/gov/household/household_state_benefits.yaml

```diff
@@ -17,6 +17,8 @@ values:
     - ca_capi
     # Nebraska benefits
     - ne_child_care_subsidies
+    # North Carolina benefits
+    - nc_scca
```

Comment:
I think we need to add this in 2024 since the subsidized rate only applies starting on 2023-10 - if we try to compute the `nc_scca` in 2023 it would break

Reply:
I added a couple of integration tests for 2023-01 and encountered no errors. To be safe, I've moved them to 2024-01 like suggested.

Reply:
uncomment